src/Internal/Contract/Monad.purs

```diff
@@ -60,11 +54,7 @@ import Ctl.Internal.QueryM
   , mkOgmiosWebSocketAff
   , underlyingWebSocket
   )
--- TODO Move/translate these types into Cardano
```

Comment:
Does this TODO have a resolution or an issue?

Reply:
Not yet, restored this TODO